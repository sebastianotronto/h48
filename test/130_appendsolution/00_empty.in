

0
1
rotation UF
