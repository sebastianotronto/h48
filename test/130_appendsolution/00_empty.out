
Number of solutions: 1
Shortest solution length: 0
Used bytes: 1
